doc/faq.rst

```diff
@@ -15,45 +15,61 @@ Why is pandas not enough?
 -------------------------
 
 pandas is a fantastic library for analysis of low-dimensional labelled data -
-if it can be sensibly described as "rows and columns", pandas is probably the
-right choice.  However, sometimes we want to use higher dimensional arrays
-(`ndim > 2`), or arrays for which the order of dimensions (e.g., columns vs
-rows) shouldn't really matter. For example, the images of a movie can be
-natively represented as an array with four dimensions: time, row, column and
-color.
-
-Pandas has historically supported N-dimensional panels, but deprecated them in
-version 0.20 in favor of Xarray data structures.  There are now built-in methods
-on both sides to convert between pandas and Xarray, allowing for more focussed
-development effort.  Xarray objects have a much richer model of dimensionality -
-if you were using Panels:
-
-- You need to create a new factory type for each dimensionality.
-- You can't do math between NDPanels with different dimensionality.
-- Each dimension in a NDPanel has a name (e.g., 'labels', 'items',
-  'major_axis', etc.) but the dimension names refer to order, not their
-  meaning. You can't specify an operation as to be applied along the "time"
-  axis.
-- You often have to manually convert collections of pandas arrays
-  (Series, DataFrames, etc) to have the same number of dimensions.
-  In contrast, this sort of data structure fits very naturally in an
-  xarray ``Dataset``.
-
-You can :ref:`read about switching from Panels to Xarray here <panel transition>`.
+if the data can be sensibly described as "rows and columns", pandas is probably the
+right choice.  However, xarray has some distinct advantages: higher dimensionality
+and an explicit, production-ready API.
```

Comment:
this sounds a bit as if the pandas API was not production-ready

Reply:
I'll soften to a comparative

Reply:
I'm not sure either xarray or pandas are production ready :)